packages/repository/package.json

```diff
@@ -25,13 +25,15 @@
     "@loopback/build": "^0.6.9",
     "@loopback/testlab": "^0.10.8",
     "@types/lodash": "^4.14.108",
+    "@types/lodash.camelcase": "^4.3.3",
     "@types/node": "^10.1.1",
     "lodash": "^4.17.10"
   },
   "dependencies": {
     "@loopback/context": "^0.11.7",
     "@loopback/core": "^0.9.0",
     "@loopback/dist-util": "^0.3.3",
+    "lodash.camelcase": "^4.3.0",
```

Comment:
What is the reasoning for using `lodash.camelcase` instead of full `lodash` module, which is already in our dev-dependencies? I understand `lodash.camelcase` is smaller than `lodash`, but then other packages are already depending on `lodash` and therefore it's very likely that `lodash` is going to be installed in the dependency tree anyways.

Thoughts?